securesystemslib/util.py

```diff
@@ -395,6 +95,34 @@ def get_file_details(filepath, hash_algorithms=['sha256']):
   return file_length, file_hashes
 
 
+def persist_temp_file(temp_file, persist_path):
+  """
+  <Purpose>
+    Copies 'temp_file' (a file like object) to a newly created non-temp file at
+    'persist_path' and closes 'temp_file' so that it is removed.
+
+  <Arguments>
```

Comment:
Would you mind adding `temp_file` to the argument section as well?

Reply:
Just found out about GitHub suggestions. :) Does this work for you?

```suggestion
  <Arguments>
  temp_file:
    A file-like object, e.g. as returned by 'tempfile.TemporaryFile()'.
     
```

Reply:
Great catch. I've added this to the first commit and force pushed an updated series.

Reply:
Hooray!

Reply:
> Just found out about GitHub suggestions. :) Does this work for you?

Apologies, I missed that. This change would have been fine.

Reply:
No worries, yours is actually better. :)